WooCommerce/WooCommerceTests/ViewRelated/Search/Order/OrderSearchUICommandTests.swift

```diff
@@ -0,0 +1,171 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+import Storage
+import Networking
+import protocol WooFoundation.Analytics
+
+final class OrderSearchUICommandTests: XCTestCase {
+    let siteID: Int64 = 12345
+    private var storageManager: MockOrderStatusesStoresManager!
+    private var analyticsProvider: MockAnalyticsProvider!
+    private var analytics: WooAnalytics!
+
+    override func setUpWithError() throws {
+        super.setUp()
+        storageManager = MockOrderStatusesStoresManager()
+        analyticsProvider = MockAnalyticsProvider()
+        analytics = WooAnalytics(analyticsProvider: analyticsProvider)
+    }
+
+    override func tearDownWithError() throws {
+        storageManager = nil
+        analyticsProvider = nil
+        analytics = nil
+        super.tearDown()
+    }
+
+    func testCreateStarterViewControllerReturnsNil() throws {
+        // Given
+        let command = OrderSearchUICommand(siteID: siteID, onSelectSearchResult: { _, _ in }, storageManager: storageManager)
+
+        // When
+        let starterViewController = command.createStarterViewController()
+
+        // Then
+        XCTAssertNil(starterViewController, "Expected createStarterViewController to return nil")
+    }
+
+    func testCreateResultsController() {
+        // Given
+        let command = OrderSearchUICommand(siteID: siteID, onSelectSearchResult: { _, _ in }, storageManager: storageManager)
+
+        // When
+        let resultsController = command.createResultsController()
+
+        // Then
+        XCTAssertNotNil(resultsController, "Expected createResultsController to return a ResultsController instance")
+        XCTAssertEqual(resultsController.predicate?.predicateFormat, "siteID == \(siteID)", "Predicate format did not match")
+        XCTAssertEqual(resultsController.sortDescriptors?.first?.key, "dateCreated", "First sort descriptor key did not match")
+        XCTAssertEqual(resultsController.sortDescriptors?.first?.ascending, false, "First sort descriptor ascending did not match")
+    }
+
+    func testCreateCellViewModel() {
+        // Given
+        let mockOrder = MockOrders().makeOrder(status: .onHold, items: [], shippingLines: [], refunds: [], fees: [], taxes: [], customFields: [], giftCards: [])
```

Comment:
Feel free to use the defaults... they should be generally safe. I've left the differences intact, but perhaps you don't even need to specify the shippingLines here?

```suggestion
        let mockOrder = MockOrders().makeOrder(status: .onHold, shippingLines: [])
```

Reply:
That's a good change since it increases clarity about the focus of the test.

Addressed in commit 7cd80ee